mcs/class/Makefile

```diff
@@ -197,10 +199,10 @@ net_4_x_dirs := \
 	System.Core			\
 	System.Security			\
 	System.Configuration	\
+	System.Drawing			\
 	$(resgen_dir)			\
```

Comment:
I don't think we can remove it as we wouldn't install the resgen from the `build` profile?